lua/vis.lua

```diff
@@ -152,6 +152,7 @@ local events = {
 	WIN_OPEN = "Event::WIN_OPEN", -- see @{win_open}
 	WIN_STATUS = "Event::WIN_STATUS", -- see @{win_status}
 	TERM_CSI = "Event::TERM_CSI", -- see @{term_csi}
+  PROCESS_RESPONCE = "Event::PROCESS_RESPONCE", -- see @{process_responce}
```

Comment:
Is responce perhaps a typo of the word response?

Reply:
It is. Thank you for noticing!